examples/rvc-app/rvc-common/src/rvc-device.cpp

```diff
@@ -104,7 +106,39 @@ void RvcDevice::HandleOpStatePauseCallback(Clusters::OperationalState::GenericOp
 
 void RvcDevice::HandleOpStateResumeCallback(Clusters::OperationalState::GenericOperationalError & err)
 {
-    auto error = mOperationalStateInstance.SetOperationalState(to_underlying(OperationalState::OperationalStateEnum::kRunning));
+    // This method is only called if the device is in a resume-compatible state, i.e. `Charging`, `Docked` or `Paused`.
```

Comment:
Place this comment in the `default` branch of the switch, log an error and set the error handle.

Reply:
Moved it.